### DISCUSSION TITLE
Possible Features for ProjectFNF!!

### DISCUSSION THREAD

**Discussion by aflacc** [aflacc/ProjectFNF#2] [Ideas]
This is the discussion board for sharing your ideas on the mod! You can share them in your own discussion or issue as well, but this is a great place to check for duplicate suggestions!

**Comment by crispyi2** [comment]
I have a few:
- A way to add a trigger for a alternate pose/animation in the in-game chart editor
- A easier way to create custom weeks/characters
- and echo support of you haven't added it yet

**Reply by crispyi2** [reply]
I like this engine more than kade engine already because of the features it will have in the first official release

**Comment by aflacc** [comment]
> A way to add a trigger for a alternate pose/animation in the in-game chart editor

I actually was thinking about this but the only thing is that I don't know how I'd do it.
> A easier way to create custom weeks/characters

There will be a whole wiki thing on it and I'm trying to add a failsafe that basically prevents certain things like healthicons from causing a crash
> and echo support of you haven't added it yet 

Could you explain what you mean by this?

Thank you for using ProjectFNF. Also, anything thats related to neo stuff should be removed. You can keep them if you want its just they should be removed. Thank you.

**Reply by crispyi2** [reply]
that last one got autocorrected sorry for the missunder standing i meant to type dfjk support for hitting notes

**Reply by aflacc** [reply]
1) ok so dfjk autocorrected to echo lol
2)  Its not hard to make it support DFJK. I'm thinking of just making that a built in thing considering I don't think anyone actually uses WASD. Also, if you have any questions, feel free to join the discord by clicking the discord badge in the readme.

**Comment by aflacc** [comment]
If you want to suggest something, create a new discussion page.
